master/internal/user/postgres_users_intg_test.go

```diff
@@ -0,0 +1,151 @@
+//go:build integration
+// +build integration
+
+package user
+
+import (
+	stdContext "context"
+	"fmt"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/etc"
+
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+func setupBun(t *testing.T) {
```

Comment:
I've run into this lately. It feels better to just add a TestMain IMO, rather than repeat this all over.